recipes/protobufc/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "protobufc" %}
```

Comment:
can we name the package `protobuf-c`? It's not the name of the binary, but it matches the upstream project name, and it distinguishes more easily from `protobuf`.

Reply:
renamed as suggested